pytradfri/api/libcoap_api.py

```diff
@@ -206,22 +221,5 @@ def _process_output(output, parse_json=True):
     if output.startswith(SERVER_ERROR_PREFIX):
         raise ServerError(output)
     if not parse_json:
-        return output
-    return json.loads(output)
-
-
-def retry_timeout(api, retries=3):
```

Comment:
I can't see that this is being used anywhere? I thus removed it. Maybe do that in a separate PR and then look at adding type hints?

Reply:
I think we can remove it. I think it was used a long time ago as a helper.